messaging/src/main/scala/uk/ac/wellcome/messaging/worker/models/Result.scala

```diff
@@ -3,7 +3,7 @@ package uk.ac.wellcome.messaging.worker.models
 sealed trait Result[Summary] {
   val summary: Option[Summary]
   def pretty =
-    s"${this.getClass.getSimpleName}: $summary"
+    s"${this.getClass.getSimpleName}: ${summary.getOrElse("no-summary")}"
```

Comment:
tiny nit: I would wrap this in a delimiter to make it clear it’s not a real message, e.g.
```suggestion
    s"${this.getClass.getSimpleName}: ${summary.getOrElse("<no-summary>")}"
```